exe/wallet/Main.hs

```diff
@@ -219,11 +233,38 @@ exec manager args
             Just version -> do
                 TIO.putStrLn $ T.pack version
 
+    | args `isPresent` command "transaction" &&
+      args `isPresent` command "create" = do
+        wId <- args `parseArg` longOption "wallet-id"
+        ts <- args `parseAllArgs` longOption "payment"
+        wPwd <- getPassphrase
+        runClient @Transaction Aeson.encodePretty $ createTransaction (ApiT wId) $
+            PostTransactionData
+                -- NOTE: we are sure this will be non-empty
```

Comment:
Side-note: for this kind of comment, the reason _why_ we are sure is really what we're interested in :wink: